soepy/exogenous_processes/children.py

```diff
@@ -17,8 +17,8 @@ def define_child_age_update_rule(model_spec, states, covariates):
     # Age stays at -1 if no kids so far
     child_age_update_rule = np.full(states.shape[0], -1)
     # Age increases by one, if there is a kid
-    child_age_update_rule[np.where(covariates[:, 1] != 0)] = (
-        states[np.where(covariates[:, 1] != 0)][:, 6] + 1
+    child_age_update_rule[np.where(covariates[:, 0] != 0)] = (
```

Comment:
How come this change was required?

Reply:
This was necessary because in the previous version the education level was generated as a covariate and took position zero in the covariates array. Now the education level is part of the state space and not the covariates object. Therefore all indices referencing covariates have been adjusted.